CRM/Core/BAO/UserJob.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC https://civicrm.org/licensing
+ */
+
+/**
+ * This class contains user jobs functionality.
+ */
+class CRM_Core_BAO_UserJob extends CRM_Core_DAO_UserJob {
+
+  /**
+   * Get the statuses for Import Jobs.
+   *
+   * @return array
+   */
+  public static function getStatuses(): array {
+    return [
+      [
+        'id' => 1,
+        'name' => 'completed',
+        'label' => ts('Completed'),
+      ],
+      [
+        'id' => 2,
+        'name' => 'scheduled',
+        'label' => ts('Scheduled'),
+      ],
+      [
+        'id' => 3,
+        'name' => 'in_progress',
+        'label' => ts('In Progress'),
+      ],
+    ];
```

Comment:
Should there be a status like `draft` -- ie it's new but not ready to launch?

Reply:
@totten quite possibly - I can add that now or just tweak that value set as I implement / figure out which options I want in practice - since it largely stands alone from the schema?

Reply:
Yeah, that's fine. For the moment, it'll be easier to add statuses than to remove them.

Other statuses that might make sense in the future: `paused` , `canceled`